extensions/amp-base-carousel/1.0/test/test-amp-base-carousel.js

```diff
@@ -295,6 +295,39 @@ describes.realWin(
       expect(buttons).to.have.length(0);
     });
 
+    it('should go to slide 0 when index is set to 0', async () => {
```

Comment:
Does this fix also affect the 1.0 version?

Reply:
@micajuine-ho I accidentally added a test for the 1.0 version, but from this old PR I think I was told it was okay to keep - https://github.com/ampproject/amphtml/pull/30605#pullrequestreview-506670143.

I can get rid of it if that's better!

Reply:
Thanks! I think it's better to keep it! Unit tests are not expensive, so the more coverage we have the better :D